util/regexp-assemble/data/942210.data

```diff
@@ -63,3 +63,27 @@ between[\s(]+\w+[\s)]*?[!=+]+[\s\d]*?[\"'`=()]
 not[\s(]+\w+[\s)]*?[!=+]+[\s\d]*?[\"'`=()]
 \|\|[\s(]+\w+[\s)]*?[!=+]+[\s\d]*?[\"'`=()]
 \&\&[\s(]+\w+[\s)]*?[!=+]+[\s\d]*?[\"'`=()]
+
+##! Added (?:\/\*.*?\*\/)?\s* to support comment block that can be used to bypass the filter
```

Comment:
The change actually don't include the comment guard. Should they?

Reply:
Maybe. I think we need to take a look at the rule, just in case.